catalogs/sources/gtfs/realtime/us-california-santa-cruz-metro-scmtd-gtfs-rt-s-a-1912.json

```
@@ -0,0 +1,13 @@
+{
+    "mdb_source_id": 1912,
+    "data_type": "gtfs-rt",
+    "entity_type": "sa",
```

Convert the entity_type to an array:
```suggestion
    "entity_type": ["sa"],
```